### PR TITLE
refactor(cozy-intent): Update import paths

### PR DESCRIPTION
**Pull request by acezard**
Make for easier use in other TS apps

### CODE REVIEW

File: packages/cozy-intent/package.json [hunk 1]
```diff
@@ -47,13 +48,11 @@
     "directory": "packages/cozy-intent"
   },
   "scripts": {
-    "build": "rm -rf ./dist && yarn build:types && babel --extensions .ts,.tsx --ignore 'src/tests','**/*.spec.tsx','**/*.spec.ts' ./src -d ./dist --copy-files --no-copy-ignored",
+    "build": "rm -rf ./dist && yarn build:types && babel --extensions .ts,.tsx --ignore '**/*.spec.tsx','**/*.spec.ts' ./src -d ./dist --copy-files --no-copy-ignored",
```
Comment (trollepierre) [inline packages/cozy-intent/package.json:51]: let's split this command 🙏 

"build:clean": "rm -rf ./dist"

Comment (trollepierre) [reply packages/cozy-intent/package.json:51]: why not using cozy-script?

Comment (acezard) [reply packages/cozy-intent/package.json:51]: cozy-script is too heavy here I think, we don't really need webpack, just babel to transpile ts to js imo

File: packages/cozy-intent/tests/mocks.ts [hunk 2]
```diff
@@ -2,10 +2,12 @@ import { ChildHandshake, RemoteHandle } from 'post-me'
...
-import { WebviewMessenger } from '../api/services/WebviewMessenger'
-import { WebviewService } from '../api/services/WebviewService'
-import { WebviewWindow } from '../api/models/environments'
-import { NativeMethodsRegister } from 'api/models/methods'
+import {
+  NativeMethodsRegister,
+  WebviewMessenger,
+  WebviewService,
+  WebviewWindow
+} from '../src/api'
```
Comment (trollepierre) [inline packages/cozy-intent/tests/mocks.ts:10]: @api ?

**Review by trollepierre — APPROVED** [review]
Well done, it helps a lot the code readability! 👏

File: packages/cozy-intent/tests/mocks.ts (hunk 2 above)
Comment (acezard) [reply packages/cozy-intent/tests/mocks.ts]: updated

File: packages/cozy-intent/package.json (hunk 1 above)
Comment (acezard) [reply packages/cozy-intent/package.json]: updated

**Pull request merged**